tests/tracking/test_rest_tracking.py

```diff
@@ -528,6 +528,21 @@ def test_validate_path_is_safe_good(path):
     validate_path_is_safe(path)
 
 
+@pytest.mark.skipif(not is_windows(), reason="This test only passes on Windows")
+@pytest.mark.parametrize(
+    "path",
+    [
+        # relative path from current directory of C: drive
+        "C:path",
+        "C:path/",
+        "C:path/to/file",
```

Comment:
Is `C:../path/to/file"` bad?

Reply:
emmm I don't think that's a valid path? 
<img width="666" alt="image" src="https://github.com/mlflow/mlflow/assets/82044803/7c203e0e-a8c6-44cd-be3e-c7370ec0f326">

Reply:
But if providing this path, it will be identified as a relative path 🤔

Reply:
good relative path or bad relative path?

Reply:
good relative path I think, it doesn't meet any of the three conditions to throw an exception